herbert_core/classes/instrument_classes/instrument/instruments/@IX_inst_DGfermi/IX_inst_DGfermi.m

```diff
@@ -70,6 +70,7 @@
         function obj=set.energy(obj,val)
             obj.moderator_.energy = val;
             obj.fermi_chopper_.energy = val;
+            obj = obj.clear_hash();
```

Comment:
I am not aware of way of doing this in Matlab. In Python it could probably be done through wrapper functions. 
In any situation, it would be some kind of global setter function used everywhere and overloaded in `hashable`  We do not have such function, so I am taking approach choosen in `serializable` for `check_common_args`

Subclasses should use common methods of parent class.  This is why parent class protected methods are for. And public too, though public may be used for other purposes too.